src/System.Net.Sockets/tests/FunctionalTests/DualModeSocketTest.cs

```diff
@@ -2715,7 +2716,7 @@ public SocketUdpClient(ITestOutputHelper output, Socket serverSocket, IPAddress
                 _port = port;
                 _serverSocket = serverSocket;
 
-                Task.Run(() => ClientSend(null, redundant));
+                _t = Task.Run(() => ClientSend(null, redundant));
```

Comment:
Is it worth some grepping for other instances in the networking tests where there is Task.Run without an attempt to wait on the task?
Eg I see a similar pattern in the SocketClient class higher up in this file.

Reply:
I was thinking about it @danmosemsft but I would like to get feedback first.
It also seems like  https://github.com/dotnet/corefx/pull/24099 did the trick - at least I have not seen failures again.